eng/spotbugs-reporting/pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>azure-client-sdk-parent</artifactId>
+        <groupId>com.azure</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.client.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spotbugs-reporting</artifactId>
+    <version>1.0.0</version>
+
+    <properties>
+        <azure-keyvault.version>1.2.0</azure-keyvault.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-keyvault</artifactId>
+            <version>${azure-keyvault.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-keyvault-cryptography</artifactId>
+            <version>${azure-keyvault.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-keyvault-core</artifactId>
+            <version>${azure-keyvault.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-keyvault-extensions</artifactId>
+            <version>${azure-keyvault.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-keyvault-complete</artifactId>
+            <version>${azure-keyvault.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-keyvault-webkey</artifactId>
+            <version>${azure-keyvault.version}</version>
+        </dependency>
+    </dependencies>
```

Comment:
Yes, 
Build helper plugin dymanically imports the source during build time.
Dependencies ensure that code compiles.
If we just invoke spotbugs:spotbugs it doesn't work, because the aggregated code only gets imported at build time.
As discussed, we will explore ways to optimize this or use open source plugins available.